applications: nrf_desktop: Fix dvfs module consuming event (#21140)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    The negative impact of USB polling jitter is more visible in case of USB High-Speed.
  * The Fast Pair sysbuild configurations to align the application with the sysbuild Kconfig changes for controlling the Fast Pair provisioning process.
    The Nordic device models intended for demonstration purposes are now supplied by default in the nRF Desktop Fast Pair configurations.

>   * The DVFS module no longer consumes the ble_peer_conn_params.

Reviewer: `ble_peer_conn_params_event`

Reviewer: Please also add doxygen link to event and reflinks to modules' doc pages

>  * The DVFS module to no longer consume the ``ble_peer_conn_params_event``.

Reviewer 2: ```suggestion
  * The :ref:`nrf_desktop_dvfs` to no longer consume the ``ble_peer_conn_params_event``.
```

    The negative impact of USB polling jitter is more visible in case of USB High-Speed.
  * The Fast Pair sysbuild configurations to align the application with the sysbuild Kconfig changes for controlling the Fast Pair provisioning process.
    The Nordic device models intended for demonstration purposes are now supplied by default in the nRF Desktop Fast Pair configurations.

>   * The :ref:`nrf_desktop_dvfs` to no longer consume the ``ble_peer_conn_params_event``.

Reviewer 2: ```suggestion
  * The :ref:`nrf_desktop_dvfs` to no longer consume the :c:struct:`ble_peer_conn_params_event`.
```
This might do the link to the event.

    This allows to propagate the event to further listeners of the same or lower priority.

>     This prevents an issue where ``ble_latency`` is not informed about the connection parameter update (it might cause missing connection latency updates).

Reviewer: ```suggestion
    This prevents an issue where :ref:`nrf_desktop_ble_latency` is not informed about the connection parameter update (it might cause missing connection latency updates).
```

Reviewer 2: Oh, I missed that this was also one of the module docs.
